docs/architecture/viam-server.md

```diff
@@ -62,6 +62,16 @@ You can also revert to an earlier configuration from the History tab.
 If you only want to apply configuration updates under certain conditions, you can create a sensor with your conditions.
 The sensor must return a true value when it is safe to update and a false value otherwise.
 
+{{< tabs >}}
+{{% tab name="Builder UI" %}}
+
+To use a fragment, click the **+** icon next to your {{< glossary_tooltip term_id="part" text="machine part" >}} in the left-hand menu of the **CONFIGURE** tab and select **Maintenance window**.
```

Comment:
does this mean to say to use a fragment?

Reply:
testing it I see its like a fragment added but the phrasing is confusing to me, lmk what you think